src/cba-list-new/shadow.css

```diff
@@ -1,4 +1,4 @@
-:host
+:host 
```

Comment:
After changes in https://github.com/browser-automation/cba-components/pull/59 get merged into `master`, we will need to remove top comment and run `npm run test:stylelint -- fix` to fix all unrelated changes here, whatever will not be covered by the automated fix manual input might be required.